elegy/losses/binary_crossentropy.py

```diff
@@ -0,0 +1,99 @@
+from elegy import types
+import typing as tp
+import jax
+import jax.numpy as jnp
+from elegy import utils
+from elegy.losses.loss import Loss, Reduction
+
+
+def binary_crossentropy(
+    y_true: jnp.ndarray, 
+    y_pred: jnp.ndarray, 
+    from_logits: bool = False
+    ) -> jnp.ndarray:
+
+    if from_logits:
+        return -jnp.mean(y_true * y_pred - jnp.logaddexp(0.0, y_pred), axis=-1)
+
+    y_pred = jnp.maximum(y_pred, utils.EPSILON)
```

Comment:
use

```python
y_pred = jnp.clip(y_pred, utils.EPSILON, 1.0 - utils.EPSILON)
```

Reply:
Does [dd998a6](https://github.com/poets-ai/elegy/pull/51/commits/dd998a6d3bca296d4be756f51e7f668c92a6a397) make it? Or is there something else missing?
Test passing.